app/src/main/java/com/kamron/pogoiv/ScreenShotHelper.java

```diff
@@ -0,0 +1,47 @@
+package com.kamron.pogoiv;
+
+import android.content.Context;
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.os.FileObserver;
+import android.support.v4.content.LocalBroadcastManager;
+
+import java.io.File;
+
+/**
+ * Created by Sarav on 9/9/2016.
+ */
+public class ScreenShotHelper {
+
+    private static ScreenShotHelper instance = null;
+    private FileObserver screenShotScanner;
+
+    private ScreenShotHelper(final Context context, final String screenShotDir) {
+        screenShotScanner = new FileObserver(screenShotDir, FileObserver.CLOSE_NOWRITE | FileObserver.CLOSE_WRITE) {
+            @Override
+            public void onEvent(int event, String file) {
+                if (file != null) {
```

Comment:
Memo: This commit stops using `readyForNewScreenshot` silently. Hopefully later commits clean this up.